### ISSUE TITLE
Tabs: Scroll buttons should be hidden for Vertical tabs

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#6360]
**Describe the issue. What is the expected and unexpected behavior?**
The scroll buttons are not hidden when the tabs are vertical.

cc @jessiehuff
